testsuite/features/core_ubuntu_salt_ssh.feature

```diff
@@ -7,21 +7,6 @@
 
```

Comment:
Bump date to 2019 in the copyright.

We want only *one* ubuntu feature in core. So this should be renamed to min_ubuntu_salt_ssh.feature, assuming that by default ubuntu is "salt" and not "salt ssh".
->
same, i suggest you commit "as is" and let @lkotek move it to non-core.